packages/e2e/features/ui/components/liveness/disable-start-screen.feature

```diff
@@ -16,3 +16,14 @@ Feature: Disable Start Screen
       Then I see the "Face didn't fit inside oval in time limit." timeout error
       Then I click the "Try again" button
       Then I see "Loading"
+
+  @react
+  Scenario: See camera module and instructions with face movement challenge
+      I see "Update Challenge Selection"
```

Comment:
Doesn't this need to be "Given I see" or "When I see"?  I've never seen Gherkin without starting with a verb
https://cucumber.io/docs/gherkin/reference/

Reply:
lol yes, updated